### ISSUE TITLE
Support integration with Apache Tomcat manager application

### ISSUE THREAD

**Issue by owenfarrell** [mkloubert/vs-deploy#69]
I'm trying to integrate the vs-deploy extension with Apache Tomcat via an HTTP target. But doing so seems to require 2 different changes:

1. The HTTP target implementation does not expand placeholders in the URL (e.g. output values or  ${workspaceRoot})
2. The HTTP target implementation needs an option to not submit the file contents in the HTTP body

**Comment by mkloubert** [comment]
@owenfarrell 

Great!

The best way is to create a separate branch and do a pull request to this repository when finished.

**Comment by owenfarrell** [comment]
I think this as simple as adding the following line to http.ts:188

`str = me.context.replaceWithValues(str);`

I'm trying to confirm that it's that simple, but I'm having issues getting the extension to run.

```
$ node ./node_modules/vscode/bin/install
Error installing vscode.d.ts: Missing VSCode engine declaration in package.json.
```

Ever seen that before?

**Comment by mkloubert** [comment]
That right. The [replaceWithValues()](https://mkloubert.github.io/vs-deploy/interfaces/_contracts_.deploycontext.html#replacewithvalues) method replaces all [placeholders](https://github.com/mkloubert/vs-deploy/wiki/values) which are defined in a [settings](https://github.com/mkloubert/vs-deploy/wiki#settings--) file,

You should try to run a simple

```bash
npm install
```

from the root directory of your checked out repository, to keep sure to install all required modules and TypeScript definition files.

**Comment by mkloubert** [comment]
@owenfarrell 

**Released with version 9.1.0**

Thank you very much for the great work! :-)

**Comment by mkloubert** [comment]
s. [Support for Tomcat Manager Integration](https://github.com/mkloubert/vs-deploy/pull/70#issuecomment-301273072)
